external/downstream/psi4fockci/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# Format borrowed from sns-mp2 CMakeLists
+
+if(${ENABLE_psi4fockci})
+    if(NOT (${CMAKE_DISABLE_FIND_PACKAGE_psi4fockci}))
+        include(FindPythonModule)
+        find_python_module(psi4fockci ATLEAST 1.0.0 QUIET)
```

Comment:
Need to adjust version number here or it'll never be found.

Also, need to add `psi4fockci` to https://github.com/psi4/psi4/blob/master/external/downstream/CMakeLists.txt#L5 b/c as-is this file never executed.

Good news is that the conda pkg looks to be found and the test passes.

Reply:
Good catch, thanks-- the version number should be updated now.

I think I added `psi4fockci` to the downstream CMakeLists.txt in a recent commit (https://github.com/psi4/psi4/pull/1855/commits/3881cb475e50a567a74d956d9c13ed56445729a5) but let me know if I've added it in the wrong place or need to change anything else.

Thanks so much for getting the conda package set up!!

Reply:
I see the downstream/CM change now -- maybe I hadn't refreshed before.